db/migrate/20240122141420_add_development_type_for_reporting_to_planning_applications.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class AddDevelopmentTypeForReportingToPlanningApplications < ActiveRecord::Migration[7.0]
+  def change
+    add_column :planning_applications, :development_type_for_reporting, :string
```

Comment:
I agree, I don't like it either but I used it how they called it. They called it development type for reporting, because what we call `application_type` they call it development type. How about just `reporting_type`?